doc/40_Configuration.md

```diff
@@ -49,39 +49,42 @@ image: node:alpine
 
 ## Exposing Ports
 
-If you want to expose any ports from your development workspace, for instance the served ports of
-your dev server, you can configure them in the `.gitpod` file:
+If you want to access services running in your workspace, e.g. a development HTTP server on port `8080`,
+you need to expose that port first. Gitpod has two means of doing that:
+  1. On-the-fly: when you start a process which listens on a port in your workspace, Gitpod will ask you
+     if you want to expose that port to the internet.
+  2. In your configuration: if you already know that you want a particular port exposed, you can configure
+     it in the `.gitpod` file and skip the extra click later on. For example:
 ```yaml
 ports:
   - port: 8080
     protocol: "http"
 ```
-Ports are being mapped to their own URLs prefixing the workspace URL with `{portnumber}-`. For
-instance: `https://8080-fe76ea5b-924d-4a67-a2d5-24a259619fa7.ws.gitpod.io/`. At the moment only the
-`http` protocol is available.
+Ports are mapped to their own URLs, prefixing the workspace URL with `{portnumber}-`. For instance:
+`https://8080-fe76ea5b-924d-4a67-a2d5-24a259619fa7.ws.gitpod.io/`. At the moment you can only expose
+HTTP servers.
 
 ## Start Script
 
-In many cases it makes sense to just start the build and maybe something like a dev server. For
-that purpose, you can provide a shell command to be run in the initial terminal on start.
+In many cases it makes sense to start the build and maybe something like a dev server upon workspace
+initialization. To that end you can provide a shell command that gets executed in a terminal on start.
 
 For instance, the start script for the Gitpod documentation repository is defined as:
 ```yaml
 tasks:
   - command: "npm install && npm run serve"
 ```
 
-You can chain multiple shell commands with `&&`.
+Similarly to Docker, you can chain multiple shell commands with `&&`.
```

Comment:
This is not so much a Docker but a bash feature

Reply:
Was thinking the same. But concluded we probably want to point out to the analogy ppl will most likely know about ...

Reply:
@32leaves ?

Reply:
It is a shell feature, but most prominently used in Docker ... at least in our context. I'm fine with any.